md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/AppDeploymentQueueStatusAPI.java

```diff
@@ -45,25 +45,25 @@
      * This method calls proc GetDeployStatus and fetches a record from DeployStatus table corresponding
      * to deployStatusId passed.
      *
-     * @param adqState
+     * @param
      * @return restWrapper It contains an instance of DeployStatus corresponding to deployStatusId passed.
      */
     @RequestMapping(value = "/{id}", method = RequestMethod.GET)
     @ResponseBody public
     RestWrapper get(
-            @PathVariable("id") Short adqState, Principal principal
+            @PathVariable("id") Short appDeploymentStatusId, Principal principal
     ) {
 
         RestWrapper restWrapper = null;
         try {
-            com.wipro.ats.bdre.md.dao.jpa.AppDeploymentQueueStatus jpaAdqStatus = appDeploymentQueueStatusDAO.get(adqState.shortValue());
+            com.wipro.ats.bdre.md.dao.jpa.AppDeploymentQueueStatus jpaAdqStatus = appDeploymentQueueStatusDAO.get(appDeploymentStatusId.shortValue());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unboxing of "appDeploymentStatusId". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2153)